### PR TITLE
Milestone 2: Update and query animals table

### PR DESCRIPTION
**Pull request by dewslyse**
In this PR, I: 

- 🐇 added a new column (`species`) to the animals table
- 🐦 inserted additional animals' data into table
- 🐎 used database transactions to update and delete records
- 🐒 answered analytical questions using aggregate functions and `GROUP BY`

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -23,3 +23,57 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 /*Animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Update and revert species column
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
Comment (uwadonat) [inline queries.sql]: - [x] Let's verify that change was made and rollbacked.
`Begin`
  update...

  Select....// to verify if change was made
`ROLLBACK`;
  Select...// to verify that change was undone

File: queries.sql [hunk 2]
```diff
@@ -23,3 +23,57 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+
+-- Update species column with 'digimon' or 'pokemon'
+BEGIN;
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
+COMMIT;
+
```
Comment (uwadonat) [inline queries.sql]: - [x] Let's verify that change persists after commit

File: queries.sql [hunk 3]
```diff
@@ -23,3 +23,57 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+-- Update species column with 'digimon' or 'pokemon'
+BEGIN;
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
```
Comment (uwadonat) [inline queries.sql]: - [x] At this query you should update where species `IS NULL`

File: queries.sql [hunk 4]
```diff
@@ -23,3 +23,57 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+UPDATE animals SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
```
Comment (uwadonat) [inline queries.sql:67]: - [x] Let's add query to delete all records in the animal table and roll back the transaction

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[dewslyse](https://github.com/dewslyse),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit and pull request messages are very descriptive ✔️
- Your code is well organized ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uche-inyama — APPROVED** [review]
### Hi @dewslyse :slightly_smiling_face: 

Your project is complete!  Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**